cmd/importer/pod/import.go

```diff
@@ -122,8 +124,8 @@ func Import(ctx context.Context, c client.Client, cache *util.ImportCache, jobs
 		return admitWorkload(ctx, c, wl)
 	})
 
-	fmt.Printf("Checked %d pods\n", summary.TotalPods)
-	fmt.Printf("Failed %d pods\n", summary.FailedPods)
+	log := ctrl.LoggerFrom(ctx)
+	log.Info("Import done", "checked", summary.TotalPods, "failed", summary.FailedPods)
 	for e, pods := range summary.ErrorsForPods {
 		fmt.Printf("%dx: %s\n\t%v", len(pods), e, pods)
```

Comment:
log.Error

Reply:
done , but with `Info`, the errors are just strings here and the additional stack trace cannot help.

Reply:
Right. `Info` should be ok too https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#using-errors-or-loggererror